ember/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mysagw",
-  "version": "0.1.2",
+  "version": "0.2.2",
```

Comment:
is there a reason for using `0.2.2` over `0.2.0`?

Reply:
There is a very good reason for that: It's early and I didn't have enough coffee yet :see_no_evil: :rofl: 

Fixed :sweat_smile: